interlay/interbtc-ui
# Fix: back button behaviour from bridge page

## Description

Fixes this [issue](https://github.com/interlay/interbtc-ui/issues/1242) which doesn't allow user to go back to previous page from bridge

## Current behaviour
Bridge page is stuck in loop when user tries to go back by clicking on browser back button


## New behaviour

When user clicks on back button, app navigates user to their previously visited page


## Reproducible testing steps:

1. Go to /bridge?tab=issue/redeem
2. Navigate to other tabs like transaction or lending
3. Go back to /bridge page
4. Navigate using browser back button -> should take you to previous pages in the correct order of stack
